gdk4-wayland/src/auto/wayland_gl_context.rs

```diff
@@ -13,6 +13,8 @@ glib::wrapper! {
     }
 }
 
+impl WaylandGLContext {}
```

Comment:
I don't know what is causing this regression...

Reply:
It's since a while, I also don't know

Reply:
I will investigate in the future. Opened an issue regarding that for now https://github.com/gtk-rs/gir/issues/1334